src/odin_pico/adapter.py

```diff
@@ -44,7 +43,7 @@ def put(self, path, request):
 
         try:
             data = json_decode(request.body)
-            logging.debug(data)
+#            logging.debug(data)
```

Comment:
General comment - if  debug log statements are no longer needed, it's good practice to remove them rather than comment out